sources/osg/TimerGPU.js

```diff
@@ -19,14 +20,17 @@ var TimerGPU = function ( gl ) {
 
     if ( gl ) {
 
-        var ext = gl.getExtension( 'EXT_disjoint_timer_query' );
+        var ext = WebGLCaps.instance().getDisjointTimerQuery();
```

Comment:
to be consistent it would be better to use the gl instance to be sure if two osgjs context exist there is no mismatch between instance.

```
WebGLCaps.instance(gl).getDisjointTimerQuery();
```